docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+arming throttles will cause the count to reset. For example, if 5 arming throttles are sent over DroneCAN, and then 1 arming throttle 
+is received over hobby PWM, the count of consecutive arming throttles will be 1. The change of source for the arming throttles caused a reset of the count. 
+
+**To summarize, the module must receive a user-configurable number of consecutive throttle commands all in the arming throttle region and 
+all from the same source to trigger an arming transition.**
+
+Arming Example
+----------------
+An example of a possible arming setup and procedure is outlined below to clarify how arming with throttle commands works.
+
+For this example, assume that the *Arm On Throttle* parameter is set to *Arm On Throttle*, the *Arm Throttle Upper Limit* is set 
+to 0.15, the *Arm Throttle Lower Limit* is set to 0.0, and *Consecutive Arming Throttles to Arm* is set to 10. The figures below show what 
+these settings look like in the IQ Control Center when applied to a module.
+
+.. figure:: ../_static/manual_images/arming/arming_example_throttle_region_parameters.png
+    :align: center
+    :width: 60%
+    :alt: Arm on Throttle and Arming Throttle Region Limits for Example
+
+    Arm on Throttle and Arming Throttle Region Limits for Example
+
+.. figure:: ../_static/manual_images/arming/arming_consecutive_throttles.png
+    :align: center
+    :width: 60%
+    :alt: Consecutive Arming Throttles to Arm for Arming Example
+
+    Consecutive Arming Throttles to Arm for Arming Example
+
+An illustration of the arming throttle region defined by these parameters is shown below to demonstrate what throttle commands will be considered arming throttles.
+
+.. figure:: ../_static/manual_images/arming/arming_example_throttle_region_2.png
+    :align: center
+    :width: 60%
+    :alt: Arming Example Throttle Regions
+
+    Throttle Region Illustration for Arming Example
+
+Now that the module is properly configured we can begin sending throttle commands to it. The module is in the disarmed state initially. Also, 
+for this example, we will assume there is only one source sending throttle commands. For more information on how multiple sources sending throttle 
+commands affect the arming process, see :ref:`arming_consecutive_throttles`.
+
+Imagine that we begin sending 30% throttle commands to the module. Nothing will happen in this case; the module will not spin and it will not arm. 
+The module is disarmed, so throttle commands will not cause it to spin. These 30% throttle commands are outside the arming throttle region, 
+so they will not cause the module to arm.
+
+If we switch to sending -10% throttle commands, we can expect the same result. The module is still disarmed, and -10% is not in the arming throttle region.
+
+Imagine instead that we begin sending 5% throttle commands. These throttle commands fall within the arming throttle region, so they are counted as arming 

```

Comment:
Now, if we begin sending 5% throttle commands, the commands fall within the arming throttle region, and will be counted as arming throttles.